src/components/About/About.tsx

```diff
@@ -1,95 +1,134 @@
-					<p>{TEXT.ABOUT_PAGE.MAIN.DESCRIPTION.P1}</p>
-				</section>
-				<section>
-					<h2>{TEXT.ABOUT_PAGE.MAIN.RESPONSIBILITIES.HEADER}</h2>
-					<ul className="responsibility-list">
-						<li>{TEXT.ABOUT_PAGE.BP_PLACEHOLDER}</li>
-						<li>{TEXT.ABOUT_PAGE.BP_PLACEHOLDER}</li>
-						<li>{TEXT.ABOUT_PAGE.BP_PLACEHOLDER}</li>
-						<li>{TEXT.ABOUT_PAGE.BP_PLACEHOLDER}</li>
-					</ul>
-					<p>{TEXT.ABOUT_PAGE.MAIN.DESCRIPTION.P1}</p>
-				</section>
-				<section>
-					<h2>{TEXT.ABOUT_PAGE.MAIN.VALUES.HEADER}</h2>
-					<ul className="value-list">
-						<li>{TEXT.ABOUT_PAGE.BP_PLACEHOLDER}</li>
-						<li>{TEXT.ABOUT_PAGE.BP_PLACEHOLDER}</li>
-						<li>{TEXT.ABOUT_PAGE.BP_PLACEHOLDER}</li>
-						<li>{TEXT.ABOUT_PAGE.BP_PLACEHOLDER}</li>
-				</ul>
-				</section>
-			</div>
-			<section className="gain-section">
-				<h2>What can you gain?</h2>
-				<div className="card-container">
-        			<Card title={TEXT.ABOUT_PAGE.GAIN_SECTION.DIRECTED_STUDIES.HEADER} desc={TEXT.ABOUT_PAGE.GAIN_SECTION.DIRECTED_STUDIES.DESCRIPTION} backgroundColor='#E0F0FF'/>
-        			<Card title={TEXT.ABOUT_PAGE.GAIN_SECTION.RESEARCH_EXP.HEADER} desc={TEXT.ABOUT_PAGE.GAIN_SECTION.RESEARCH_EXP.DESCRIPTION} backgroundColor='#E0FDFF'/>
-        			<Card title={TEXT.ABOUT_PAGE.GAIN_SECTION.GRADUATE_STUDENTS.HEADER} desc={TEXT.ABOUT_PAGE.GAIN_SECTION.GRADUATE_STUDENTS.DESCRIPTION} backgroundColor='#E0E9FF'/>
-        			<Card title={TEXT.ABOUT_PAGE.GAIN_SECTION.GLOBAL_RESEARCH.HEADER} desc={TEXT.ABOUT_PAGE.GAIN_SECTION.GLOBAL_RESEARCH.DESCRIPTION} backgroundColor='#D9F6FF'/>
-      			</div>
-			</section>
-			<div className="contact">
-				<div className='map-container'>
-					<img src={kennyMap} alt="map" />
-				</div>
-				<div className="contact-info">
-					<h2>Contact Us</h2>
-					<p>We are located in the <a href="https://psych.ubc.ca/" target="_blank">Department of Psychology</a> at the <a href="https://www.ubc.ca/" target="_blank">University of British Columbia.</a></p>
-					<p><strong>Address</strong> <br/>
-						<p className="lab-address">
-						{TEXT.LAB_INFO.ADDRESS_LINE_ONE} <br/>
-						{TEXT.LAB_INFO.ADDRESS_LINE_TWO}</p> 
-					</p>
-					<p><strong>Tel</strong> {TEXT.LAB_INFO.TEL} <br/>
-					<strong>Fax</strong> {TEXT.LAB_INFO.FAX}</p>
-					<p><strong>Contact</strong></p>
-					<ul>
-						<li><strong>Lab:</strong> {TEXT.LAB_INFO.EMAIL}</li>
-						<li><strong>Dr. Rensink:</strong> {TEXT.LAB_INFO.DRRENSINK_CONTACT}</li>
-						<li><strong>Daniel (Admin):</strong></li>
-						<li><strong>Sara (Admin):</strong></li>
-						<li><strong>Sogol (Admin):</strong></li>
-					</ul>
-					<p><strong>Lab Hours</strong> <br/>
-					Weekdays: 9AM-5PM <br/>
-					Weekends: Closed</p>		
-				</div>
-			</div>
-			<MissionStatement />
-		</div>
-	);
+const About: React.FC<AboutProps> = (props) => {
+  return (
+    <div className="About">
+      <div className="header">
+        <p className="header-text">About the Lab</p>
```

Comment:
How about changing this to: 

```suggestion
        <p className="header-text">About the Visual Cognition Lab</p>
```

Just a suggestion for your consideration 😁